tests/core/system-settings/task.yaml

```
@@ -0,0 +1,22 @@
+summary: System settings work
+
+prepare: |
+    hostname > hostname.bak
+    cp /etc/timezone timezone.bak
+
+restore: |
+    hostnamectl set-hostname "$(cat hostname.bak)" || true
+    timedatectl set-timezone "$(cat timezone.bak)" || true
+
+execute: |
+    echo "Check that setting the hostname works"
+    hostnamectl set-hostname coole-kiste
+    MATCH coole-kiste < /etc/hostname
+    hostname | MATCH coole-kiste
+
+    echo "Check that setting the timezone works"
+    timedatectl set-timezone "Europe/Berlin"
+    date +"%Z" | MATCH CEST
```

Otherwise it will fail half of the year.
```suggestion
    date +"%Z" | MATCH 'CE[S]?T'
```